### PR TITLE
COV-52: add navbar and implement sign out

### PR DESCRIPTION
**Pull request by Kastor14**
Navbar implemented and users can signout.

### CODE REVIEW

**Review by Rafistan — APPROVED** [review]

File: client/src/views/pages/Home.js [hunk 1]
```diff
@@ -1,7 +1,35 @@
...
+import { useDispatch } from "react-redux";
+import { useHistory } from "react-router-dom";
+
+function Home() {
+    const dispatch = useDispatch();
+    const history = useHistory();
+
+    const signOut = () => {
+        dispatch(handleLogout());
+        history.push("./login");
```
Comment (jason-gerard) [inline client/src/views/pages/Home.js]: I dont think you want a period before the route. Its a URL not a file. Same for below.

```suggestion
        history.push("/login");
```

Comment (Kastor14) [reply client/src/views/pages/Home.js]: I see thanks!

**Comment by Rafistan** [comment]
Screen Shot 2022-01-29 at 5 46 30 PM
Screen Shot 2022-01-29 at 5 48 31 PM

**Review by domsec — CHANGES_REQUESTED** [review]
The UI doesn't follow the mockups in Figma.

**Review by domsec — APPROVED** [review]
Interface now matches UI mockups

**Review by jason-gerard — APPROVED** [review]

**Pull request merged**